java.exercises/src/test/XPATH.txt

```diff
@@ -0,0 +1,10 @@
+XPATH
+https://demoqa.com
+
+1. $x("//input[@id='userName']")
+2. $x("//button[@class='btn btn-primary']")
+3. $x("//div[text()='Vega']")
```

Comment:
Narrow the context
//div[@class='ReactTable -striped -highlight']//div[text()='Vega']